### DISCUSSION TITLE
Please improve the Chinese translation

### DISCUSSION THREAD

**Discussion by Ktyf37** [community/community#108601] [Discussions]
![IMG_20240217_155404.jpg](https://github.com/community/community/assets/135925971/eb9a1363-ed3d-417a-ab0e-84733a20649d)

![IMG_20240217_160259.jpg](https://github.com/community/community/assets/135925971/d1ced10c-47cf-4d78-9381-6f6114312fb3)

As shown in the picture:

***天**
***小时**

But in fact it should be correctly translated as:

***天前**
***小时前**

This is obviously contradictory to the Chinese language system

**Comment by Unmesh100** [comment]
I understand you're reporting an issue with Chinese time translations, specifically that relative time expressions are missing the "前" (meaning "ago") suffix which is required in Chinese.
Let's break this down:

Current output:


***天**** (X days)
***小时** (X hours)


Correct format should be:


***天前**** (X days ago)
***小时前** (X hours ago)

You're absolutely right - the current translation is grammatically incorrect in Chinese. The word "前" (ago) is necessary when expressing relative time in Chinese, as simply stating the number of days or hours without it is incomplete and unnatural.
Would you be able to:

Share which specific GitHub page/feature you're seeing this on?
Provide any additional examples of incorrect time translations you've noticed?
